win32ss/user/user32_vista/dpi.c

```diff
@@ -0,0 +1,15 @@
+#define WIN32_NO_STATUS
+#define _INC_WINDOWS
+#define COM_NO_WINDOWS_H
+#define NTOS_MODE_USER
+#include <windef.h>
+#include <winuser.h>
+
+/*
+ * @stub
+*/
+UINT WINAPI GetDpiForWindow(HWND hWnd)
+{
+    UNREFERENCED_PARAMETER(hWnd);
+    return USER_DEFAULT_SCREEN_DPI;
```

Comment:
Returning a hardcoded value like this just makes me sad. ROS does not support GetDpiForMonitor yet either so my suggestion is: `return GetDpiForSystem()` here and implement GetDpiForSystem by calling GetDC(NULL)+GetDeviceCaps(hDC, LOGPIXELSY).

Reply:
My issue with this is that as this is written currently, it is very clearly a stub. Your suggestion makes it more difficult to notice that the function is still a stub.

Reply:
`return GetDpiForSystem();` would still look like a stub no? Before Windows 10 there was only per-monitor DPI and since ROS is not multimon this implementation would technically be correct.

Reply:
I get what you're saying now, thanks for your suggestion. I will add this.

Reply:
I changed my mind, I don't think we need to add this right now. ReactOS doesn't even support the "Large Fonts (120 DPI)" setting at all.

Reply:
Added